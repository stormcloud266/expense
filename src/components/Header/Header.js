import React from 'react'
import { NavLink } from 'react-router-dom'

import styles from './styles.module.css'

const Header = () => (
  <header>
    <h1>Expenses</h1>
    <nav>
      <NavLink to="/" activeClassName="is-active" exact={true}>Home</NavLink>
      <NavLink to="/create" activeClassName="is-active" exact={true}>Create</NavLink>
      <NavLink to="/help" activeClassName="is-active" exact={true}>Help</NavLink>
    </nav>
  </header>
)

export default Header