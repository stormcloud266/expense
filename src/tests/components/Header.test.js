import ReactShallowRenderer from 'react-test-renderer/shallow'
import React from 'react'
import Header from '../../components/Header/Header'

test('should render Header', () => {
	const renderer = new ReactShallowRenderer()
	renderer.render(<Header />)
	expect(renderer.getRenderOutput()).toMatchSnapshot()
})
