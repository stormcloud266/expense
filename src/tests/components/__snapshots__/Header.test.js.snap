// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header 1`] = `
<header>
  <h1>
    Expenses
  </h1>
  <nav>
    <NavLink
      activeClassName="is-active"
      exact={true}
      to="/"
    >
      Home
    </NavLink>
    <NavLink
      activeClassName="is-active"
      exact={true}
      to="/create"
    >
      Create
    </NavLink>
    <NavLink
      activeClassName="is-active"
      exact={true}
      to="/help"
    >
      Help
    </NavLink>
  </nav>
</header>
`;
